docs/installation.md

```diff
@@ -1,24 +1,24 @@
 # Installing Dusty
 
 ## Requirements
+
 You must have the following installed in order to run Dusty:
 
- * [Virtualbox](https://www.virtualbox.org/wiki/VirtualBox)
- * [boot2docker](http://boot2docker.io/)
+ * [VirtualBox](https://www.virtualbox.org/wiki/VirtualBox)
+ * [Docker Machine](https://docs.docker.com/machine/)
```

Comment:
I thought splitting it out this way made sense for people who don't want to use Toolbox